module/VuFind/src/VuFind/Content/Covers/PluginManager.php

```diff
@@ -74,9 +75,7 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
      * @var array
      */
     protected $factories = [
-        Amazon::class => AmazonFactory::class,
```

Comment:
This should have been deleted long ago; just cleaning it up along with other changes.